docs/working-with-projects/working-with-images/managing-helm-charts.md

```diff
@@ -150,3 +152,54 @@ helm install --ca-file=ca.crt --username=admin --password=Passw0rd --version 0.1
 
 For other more helm commands like how to sign a chart, please refer to the [helm doc](https://docs.helm.sh/helm/#helm).
 
+
+## Manage Helm Charts with the OCI-compatible registry of Harbor
+
+Helm 3 support to pull chart from a OCI-compatible registry, and push chart to a OCI-compatible registry. About how to install Helm CLI, please refer [install helm](https://helm.sh/docs/intro/install/). Run command `helm version` to make sure the version of Helm CLI is v3.0.0+.
+
+```sh
+helm version
+
+#version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}
+```
+
+### Login to the OCI-compatible registry of Harbor
+
+Before pull/push helm charts with the OCI-compatible registry of Harbor, Harbor should be logged with `helm registry login` command.
+
+```sh
+helm registry login xx.xx.xx.xx
+```
+
+{{< note >}}
+The CA file used by the Harbor is necessary to be trusted in the system due to an issue in Helm.
+{{< /note >}}
+
+### Push Charts to the artifact Repository with the CLI
+
+After logged in and before push a chart to the OCI-compatible registry of Harbor, run the `helm chart save` command to save a chart directory which will prepare the artifact for the pushing.
+
+```sh
+helm chart save dummy-chart xx.xx.xx.xx/library/dummy-chart
+```
+
+
+When the chart was saved run the `helm chart push` command to push your charts:
+
+```sh
+helm chart push xx.xx.xx.xx/library/dummy-chart:version
+```
+
+### Pull Charts from the artifact Repository with the CLI
+
+To pull charts from the the OCI-compatible registry of Harbor, run the `helm chart pull` command just like to pull image via docker cli.
+
+```sh
+helm chart pull xx.xx.xx.xx/library/dummy-chart:version
+```
+
+### Manage Helm Charts artifacts in Harbor Interface
+
+The charts pushed to the OCI-compatible registry of Harbor is treated the same as other types of artifacts. We can list, copy, delete, update labels, get details, add or remove tags for them.
```

Comment:
'The charts pushed to the OCI-compatible registry of Harbor are treated like any other type of artifact. We can list, copy, delete, update labels, get details, add or remove tags for them just like we can for container images.'